### PR TITLE
To enable working with members authority

### PR DESCRIPTION
**Pull request by pyama86**
ref: https://github.com/yaocloud/yao/pull/51

When the pull-request is adopted, it will work.

### CODE REVIEW

File: lib/kakine/config.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def self.validate_config
     def self.setup_yao
       Yao.configure do
         auth_url    @@config['auth_url']
-        tenant_name @@config['tenant_name']
```
Comment (hanazuki) [inline lib/kakine/config.rb:49]: Now that `@@config['tenant_name']` is not used, other routines related to `tenant_name` in this file can be deleted.

Comment (pyama86) [inline lib/kakine/config.rb:49]: oh..was careless
Thanks for your insight!

**Comment by hsbt** [comment]
Travis CI is failing now. ref https://travis-ci.org/yaocloud/kakine/jobs/133925663

Should we add specific version of `Yao` gem to kakine.gemspec?

**Comment by hanazuki** [comment]
Looks good and works fine with ConoHa!

**Comment by pyama86** [comment]
SaikoDa!

**Pull request merged**